### ISSUE TITLE
Latest tasks are giving me grief with S3 regions

### ISSUE THREAD

**Issue by atrauzzi** [aws/aws-vsts-tools#148]
Just in the last bit, the "Create Elastic Beanstalk Version" task has started giving me grief when I try to create a new version with files stored in a bucket outside of the region of my application.

**Comment by atrauzzi** [comment]
Note: This has literally broken all of my deployment tasks.

**Comment by steven-cuthill-otm** [comment]
can you give us more info ?

**Comment by atrauzzi** [comment]
What do you need to know?

**Comment by steveataws** [comment]
There's a note in the api documentation for [CreateApplicationVersion](https://docs.aws.amazon.com/elasticbeanstalk/latest/api/API_CreateApplicationVersion.html) for the SourceBundle data that states the bucket you use must exist in the same region as the application. This is the api the task is using under the hood.

**Comment by atrauzzi** [comment]
Up until now I was able to do it without...

**Comment by steveataws** [comment]
There hasn't been any update to the tools so far this year, so I'm unsure why it has suddenly started to fail for you. Can you share some insight into how the tasks are configured? Did you change which buckets you were using in some way?

**Comment by atrauzzi** [comment]
No, I didn't make a single change, and multiple projects of mine just stopped working, suddenly requiring that the repos EB gets its versions from match the region of the EB application I was creating a version for.

I have logs of countless deploys working prior to this requirement coming up.
